src/config.h.cmake

```diff
@@ -14,7 +14,7 @@
 # define ANTARES_VERSION_YEAR         @ANTARES_VERSION_YEAR@
 
 //! Version in CString format
-# define ANTARES_VERSION_STR          "@ANTARES_VERSION_HI@.@ANTARES_VERSION_LO@.@ANTARES_VERSION_REVISION@"
+# define ANTARES_VERSION_STR          "@ANTARES_VERSION_HI@.@ANTARES_VERSION_LO@.@ANTARES_VERSION_REVISION@@ANTARES_VERSION_TAG@"

```

Comment:
There are 2 sources files where **ANTARES_VERSION_STR** (containing a -rcX for a release candidate version) is used and an extra `rc + ANTARES_RC` is added : 
- src\ui\simulator\windows\startupwizard.cpp
- src\ui\simulator\windows\about\aboutbox.cpp

This prints **8.3.0-rc1-rc1**